docs/framework/install/guide-for-developers.md

```diff
@@ -11,188 +11,186 @@ ms.assetid: daf9d9d5-84ac-4bd9-a864-27665ffd0f5c
-  
-1.  Open the language pack download page for the .NET Framework version you've installed:  
-
-    - [.NET Framework 4.7.2 language packs](http://go.microsoft.com/fwlink/?LinkID=863258) 
-  
-    - [.NET Framework 4.7.1 language packs](http://go.microsoft.com/fwlink/?LinkID=852090) 
-
-    - [.NET Framework 4.7 language packs](http://go.microsoft.com/fwlink/?LinkID=825306) 
-
-    - [.NET Framework 4.6.2 language packs](http://go.microsoft.com/fwlink/?LinkID=780604)  
-  
-    - [.NET Framework 4.6.1 language packs](http://go.microsoft.com/fwlink/?LinkID=671747)  
-  
-    - [.NET Framework 4.6 language packs](http://go.microsoft.com/fwlink/?LinkID=528314)  
-  
-    - [.NET Framework 4.5.2 language packs](http://go.microsoft.com/fwlink/?LinkId=397701)  
-  
-    - [.NET Framework 4.5.1 language packs](http://go.microsoft.com/fwlink/?LinkId=322101)  
-  
-    - [.NET Framework 4.5 language packs](http://go.microsoft.com/fwlink/p/?LinkId=245451)  
-  
-2.  In the language list, choose the language you want to download, and wait a few seconds for the page to reload in that language.  
-  
-3.  Choose **Download**.  
-  
-The following table lists the supported languages.  
-  
+> The language packs don't contain the .NET Framework components that are required to run an app, so you must run the web or offline installer before you install a language pack. If you have already installed a language pack, uninstall it, install the .NET Framework, and then reinstall the language pack.
+
+1.  Open the language pack download page for the .NET Framework version you've installed:
+
+    - [.NET Framework 4.7.2 language packs](http://go.microsoft.com/fwlink/?LinkID=863258)
+
+    - [.NET Framework 4.7.1 language packs](http://go.microsoft.com/fwlink/?LinkID=852090)
+
+    - [.NET Framework 4.7 language packs](http://go.microsoft.com/fwlink/?LinkID=825306)
+
+    - [.NET Framework 4.6.2 language packs](http://go.microsoft.com/fwlink/?LinkID=780604)
+
+    - [.NET Framework 4.6.1 language packs](http://go.microsoft.com/fwlink/?LinkID=671747)
+
+    - [.NET Framework 4.6 language packs](http://go.microsoft.com/fwlink/?LinkID=528314)
+
+    - [.NET Framework 4.5.2 language packs](http://go.microsoft.com/fwlink/?LinkId=397701)
+
+    - [.NET Framework 4.5.1 language packs](http://go.microsoft.com/fwlink/?LinkId=322101)
+
+    - [.NET Framework 4.5 language packs](http://go.microsoft.com/fwlink/p/?LinkId=245451)
+
+2.  In the language list, choose the language you want to download, and wait a few seconds for the page to reload in that language.
+
+3.  Choose **Download**.
+
+The following table lists the supported languages.
+
 | Language              | Culture |
 | --------------------- | :-----: |
 | Arabic                | ar      |
 | Czech                 | cs      |
 | Danish                | da      |
 | Dutch                 | nl      |
 | Finnish               | fi      |
+| English (USA)         | en-US   |
```

Comment:
I moved this because I thought people would search alphabetically for "E" for English, not "U".